autodistill/detection/classification_ontology.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Any, List, Tuple
+
+from autodistill.core import Ontology
+
+@dataclass
+class ClassificationOntology(Ontology):
```

Comment:
Is this the same as a CaptionOntology?

Reply:
Yeah. I have removed the `ClassificationOntology` since it is duplicative.